websites/docs/src/content/docs/en/deployment/ansible_deployment.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Deployment using Ansible
+description: This guide provides a step-by-step process for setting up and running the Ballerine stack using Ansible.
+---
+
+import PackageManagersTabs
+  from '../../../../components/PackageManagersTabs/PackageManagersTabs.astro';
+import CodeBlock from '../../../../components/CodeBlock/CodeBlock.astro';
+
+### Installation steps on a remote virtual machine
+
+We recommend installation using Ansible
+
+1. **Install Ansible**: We recommend installing Ansible with apt:
+<CodeBlock lang={`shell`}  code={`sudo apt install -y ansible`}/>
+
+2. **Clone the project**: Use Git to clone the Ballerine repository to your local machine:
+<CodeBlock lang={`shell`}  code={`git clone https://github.com/ballerine-io/ballerine.git && cd ballerine`}/>
+
+3. **Switch to the dev branch**: After cloning, switch to the development branch:
+<CodeBlock lang={`shell`}  code={`git checkout dev`}/>
+
+4. **Navigate to ballerine_playbook directory**:
+<CodeBlock lang={`shell`}  code={`cd deploy/ansible/ballerine_playbook`}/>
+
+5. **Create inventory file**:
+<CodeBlock lang={`shell`}  code={`touch inventory.txt`}/>
+
+Now, with your editor, open the file and add the hostname or FQDN of the server(s) you want to deploy Ballerine to with the following pattern:
+
+6. **Add entries into the inventory file**:
+<CodeBlock lang={`shell`}  code={`all ansible_host={{ SERVER_HOST }} ansible_port={{ SERVER_PORT }} ansible_user={{ SERVER_USER }}`}/>
+
+If you are using SSH keypairs for authenticating your SSH connections to your server. You can tell Ansible your ssh private key file in the `inventory` file
+using `ansible_ssh_private_key_file`
+
+<CodeBlock lang={`shell`}  code={`all ansible_host={{ SERVER_HOST }} ansible_port={{ SERVER_PORT }} ansible_user={{ SERVER_USER }} ansible_ssh_private_key_file={{ SSH_PRIVATE_KEY_FILE }}`}/>
+
+After you completed the above step then we're pretty much done with the inventory
+
+
+### Start Ballerine
+
+**Run the Ansible playbook**:
+
+After complete the above step. Now the only remain step we need to do is run the Ansible playbook.
+You can run the Ansible playbook with the following command
+
+<CodeBlock lang={`shell`}  code={`cd ballerine/deploy/ansible/ballerine_playbook;
+ansible-playbook -i inventory.txt ballerine-playbook.yml`}/>
+
+The default username and password for the backoffice are:
+
+The collection flow on <CodeBlock lang={`shell`} code={`http://localhost:5137`}/>
+
+**Username:**
+<CodeBlock lang={`shell`}  code={`admin@admin.com`}/>
+
+**Password:**
+<CodeBlock lang={`shell`}  code={`admin`}/>
+
+
+## Ballerine on HTTPS
+
+### Prerequisites
+
+Before configuring SSL for your custom domain, make sure you have the following:
+
+* A domain name - You can get a custom domain from popular providers like GoDaddy, Amazon Route 53, Digital Ocean, NameCheap, and Domain.com.
+* Ports 80 and 443 are open and accessible.
```

Comment:
I think this block can be condensed and be put inline at `vite_api_url`. Something along the lines of "Incase you want to deploy ballerine on a remote server and run it on HTTPS. Note you need to own a domain and that ports 80 and 443 are forwarded".